models/meshmodel/core/policies/rego_policy_relationship.go

```diff
@@ -51,24 +51,44 @@ func NewRegoInstance(policyDir string, regManager *registry.RegistryManager) (*R
 	}, nil
 }
 
+// CustomPrintHook implements the print.Hook interface
+type CustomPrintHook struct {
+	Messages []string
+}
+
+// Print captures print messages from policy evaluation
+// Implements print.Hook interface
+func (h *CustomPrintHook) Print(ctx print.Context, s string) error {
+	h.Messages = append(h.Messages, s)
+	logrus.Info("[OPA] ", s)
```

Comment:
Should MeshKit use its own logger?

Reply:
yeah  ,  right now there is no instance of meshkit logger  present in meskit  
we should probably configure a instance of logger in meskit and use it .